### DISCUSSION TITLE
I'm newbie, I would like to contribute to PMD project.

### DISCUSSION THREAD

**Discussion by PrasannaJatla** [pmd/pmd#4935] [General]
Hi team, I'm newbie, I would like to contribute to PMD project. This is my first time contributing to open source. I'm not sure where to start. Can some one please guide me.

**Comment by jsotuyod** [comment]
First of all, welcome!

For starters, clone the repo and ensure to import it to your favorite IDE. A guideline for Eclipse and IntelliJ is [available on our Wiki](https://github.com/pmd/pmd/wiki/Setup-IDE). It's mostly up to date, except you can safely ignore everything mentioning `apex-jorje`, that is no longer part of PMD.

Once that's done, pick an issue! If you don't have anything particular in mind, pick one from [our tracker](https://github.com/pmd/pmd/issues?q=is%3Aopen+is%3Aissue). *Usually*, [documentation bugs](https://github.com/pmd/pmd/issues?q=is%3Aopen+is%3Aissue+label%3Ain%3Adocumentation), [false positives ](https://github.com/pmd/pmd/issues?q=is%3Aopen+is%3Aissue+label%3Aa%3Afalse-positive)and [false negatives](https://github.com/pmd/pmd/issues?q=is%3Aopen+is%3Aissue+label%3Aa%3Afalse-negative) are the easiest to fix, in that order. If you pick one, leave a comment so we are aware and may assist you further.
